lib/Common/CMakeLists.txt

```diff
@@ -9,3 +9,13 @@ add_subdirectory (DataStructures)
 add_subdirectory (Exceptions)
 add_subdirectory (Memory)
 add_subdirectory (Util)
+
+
+if(NOT CC_XCODE_PROJECT)
+    add_custom_target(ChakraCoreHeaders ALL
```

Comment:
You may copy all the header files from `lib/CMakeLists.txt` Header file locations are relative to `${CMAKE_CURRENT_SOURCE_DIR}` anyway.